gnovm/pkg/gnolang/helpers.go

```diff
@@ -7,6 +7,34 @@ import (
 	"strings"
 )
 
+// ----------------------------------------
+// Functions centralizing definitions
+
+// RealmPathPrefix is the prefix used to identify pkgpaths which are meant to
+// be realms and as such to have their state persisted. This is used by [IsRealmPath].
+const RealmPathPrefix = "gno.land/r/"
+
+// ReGnoRunPath is the path used for realms executed in maketx run.
+// These are not considered realms, as an exception to the RealmPathPrefix rule.
+var ReGnoRunPath = regexp.MustCompile(`^gno\.land/r/g[a-z0-9]+/run$`)
```

Comment:
I'm not sure if we should be "mentioning" chain logic in the actual vm package

Reply:
I fully realize we moved this over from `realm.go`

Reply:
Perdoname Milosa por mi vida loca, but the whole three functions added in this file should likely be moved out of gno.land; decoupling the logic to understand "is this package a realm" so we can manage it outside the VM

But I'll leave it for another time; quite honestly. I think making sure that all our clusterfucks are in one place is already an improvement that can help us remove them eventually